tools/mqppep/repository_dependencies.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
```

Comment:
this file can be removed

Reply:
@bgruening I thought that this was required for `suite_mqppep`.  Is that not so? That's how I interpreted [this (possibly out-of-date) reference.
](https://planemo.readthedocs.io/en/latest/standards/docs/best_practices/repositories.html?highlight=suite#suites)

Reply:
@eschen42 Testing in a local v22.01 shed, this file is generated automatically, which probably answers and addresses this question.

Reply:
@bgruening Indeed, when I deleted this file I still got `suite_mqppep` in the toolshed.  So, this is fine.